### PR TITLE
New features

### PR DESCRIPTION
**Pull request by AhmedNazirMusah**
Pull request into main from my new branch called "new-features". On this new-features branch I added the following:
Completed the linter setup for HTML and CSS, added a .gitignore file, added a .html file that includes a header with the text "Hello Microverse!", added a .css file that changes the header color to green, and added a test.md file with the text "This file should be ignored by git".

### CODE REVIEW

File: .github/workflows/.hintrc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "formatters": ["stylish"],
+    "hints": [
+      "button-type",
+      "disown-opener",
+      "html-checker",
+      "meta-charset-utf-8",
+      "meta-viewport",
+      "no-inline-styles:error"
+    ]
+  }
```
Comment (Abenezer-Tilahun) [inline .github/workflows/.hintrc]: - When you copy [.hintrc](https://github.com/microverseinc/linters-config/blob/master/html-css/.hintrc) it must be in the root directory of your project not inside github/workflow folder. You can checkout configuration [here](https://github.com/microverseinc/linters-config/tree/master/html-css). Please fix it.

File: README.md [hunk 2]
```diff
@@ -1 +1,40 @@
...
+- Make sure that the last file is actually ignored by git and is not present in your GitHub repository.     
+
+
+## Authors
+
+👤 Ahmed Nazir Musah
+
+- GitHub: [@githubhandle](https://github.com/ahmednazirmusah)
+- Twitter: [@twitterhandle](https://twitter.com/ahmednazirmusah)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ahmed-nazir-musah-529956214?lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_view_base_contact_details%3BaN%2F7ouJiQmaaZdym2t1Uxg%3D%3D)
```
Comment (Abenezer-Tilahun) [inline README.md]: [OPTIONAL] Please try to update the README file, README file should be descriptive and all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in your Repo.

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @AhmedNazirMusah  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

✔️ No linter errors.
✔️ You used correct Github Flow.

almost

## Required Changes ♻️
_Check the comments below the review._

- [ ] README file should be descriptive and  all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in your Repo. 
- [ ] Please try to use meaningful Commit messages in your feature project.(at this moment you are not required to update the commit messages)
- [ ] Pull Request should have a descriptive title telling you what changes are inside and should have a summary describing introduced changes in more detail.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/.hintrc (hunk 1 above)
Comment (AhmedNazirMusah) [reply .github/workflows/.hintrc]: Thanks for your feedback. I have changed the location of the file in the new commit kindly verify if I am on the right track.

File: README.md (hunk 2 above)
Comment (AhmedNazirMusah) [reply README.md]: I have updated the Readme file as well. Thank you

**Review by BertrandConxy — APPROVED** [review]
Hi @AhmedNazirMusah , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 
- no linter errors
- good PR description
- good README
- good communication
 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AhmedNazirMusah** [comment]
Thanks for your time, I really appreciate it.

**Pull request merged**